src/app/zap-templates/zcl/data-model/chip/camera-av-settings-user-level-management-cluster.xml

```diff
@@ -0,0 +1,125 @@
+    <client init="false" tick="false">true</client>
+    <features>
+      <feature bit="0" code="DPTZ" name="DigitalPTZ" summary="Digital PTZ support">
+        <optionalConform/>
+      </feature>
+      <feature bit="1" code="MPAN" name="MechanicalPan" summary="Mechanical Pan support">
+        <optionalConform choice="a" more="true" min="1"/>
+      </feature>
+      <feature bit="2" code="MTILT" name="MechanicalTilt" summary="Mechanical Tilt support">
+        <optionalConform choice="a" more="true" min="1"/>
+      </feature>
+      <feature bit="3" code="MZOOM" name="MechanicalZoom" summary="Mechanical Zoom support">
+        <optionalConform choice="a" more="true" min="1"/>
+      </feature>
+      <feature bit="4" code="MPRESETS" name="MechanicalPresets" summary="Mechanical saved presets support">
+        <optionalConform>
+          <orTerm>
+            <feature name="MPAN"/>
+            <feature name="MTILT"/>
+            <feature name="MZOOM"/>
+          </orTerm>
+        </optionalConform>
+      </feature>
+    </features>
+    <server init="false" tick="false">true</server>
+    <globalAttribute code="0xFFFD" side="either" value="1"/>
+    <attribute code="0x0000" side="server" define="MPTZ" type="MPTZStruct" optional="true">MptzPosition</attribute>
+    <attribute code="0x0001" side="server" define="MAX_PRESETS" type="int8u" optional="true" default="5">MaxPresets</attribute>
+    <attribute code="0x0002" side="server" define="PRESET_MPTZ_TABLE" type="array" entryType="MPTZPresetStruct" optional="true">MptzPresets</attribute>
+    <attribute code="0x0003" side="server" define="MPTZ_RELATIVE_MOVE" type="array" optional="true" entryType="int16u">DptzRelativeMove</attribute>
+    <attribute code="0x0004" side="server" define="DPTZ_RELATIVE_MOVE" type="int8u" optional="true" min="2" max="100" default="100">ZoomMax</attribute>
+    <attribute code="0x0005" side="server" define="TILT_MIN" type="int16s" min="-180" max="179" default="-90" optional="true">TiltMin</attribute>
+    <attribute code="0x0006" side="server" define="TILT_MAX" type="int16s" min="-179" max="180" default="90" optional="true">TiltMax</attribute>
+    <attribute code="0x0007" side="server" define="PAN_MIN" type="int16s" min="-180" max="179" default="-180" optional="true">PanMin</attribute>
+    <attribute code="0x0008" side="server" define="PAN_MAX" type="int16s" min="-179" max="180" default="180" optional="true">PanMax</attribute>
+    <command code="0x00" source="client" name="MptzSetPosition" optional="true">
+      <description>This command SHALL set the values for the pan, tilt, and zoom in the mechanical PTZ.</description>
+      <arg id="0" name="Pan" type="int16s" optional="true"/>
+      <arg id="1" name="Tilt" type="int16s" optional="true"/>
+      <arg id="2" name="Zoom" type="int8u" optional="true" min="1" max="1"/>
```

Comment:
So the way this is specced, it's allowed to send a Pan value (but no other values) to a server that does not support MPAN.  Is that intended?  Similar for the other fields here.

This applies to some of the other commands too.

In particular, seems to me like instead of `O.b+` the conformance for Pan should be `[MPAN].b+` or something?  Need to check things carefully.

Reply:
Please see draft spec PR to address this (and other comments)
https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/10712